--- a/browser/base/content/browser-siteProtections.js
+++ b/browser/base/content/browser-siteProtections.js
@@ -1553,7 +1553,7 @@
       this._trackingProtectionIconContainer.hidden = true;
       return;
     }
-    this._trackingProtectionIconContainer.hidden = false;
+    this._trackingProtectionIconContainer.hidden = true;
 
     // Check whether the user has added an exception for this site.
     this.hasException = ContentBlockingAllowList.includes(
